draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1221,7 +1236,7 @@ minimum time for the client to wait before retrying the request.
 
 ## Submit Entry to Log {#submit-entry}
 
-POST https://{host}/.well-known/ct/v2/{prefix}/submit-entry
+POST {`submit-entry URL`}
```

Comment:
Nit: It would be nice to add an example URL (e.g. https://log.example.com/ct/v2/submit-entry) so we're more likely to end up with a unified log url organization.

Reply:
I didn't include any examples because I was taking "under-specifying it right now is the only option" (to quote your message on the mailing list) to what I thought was its logical conclusion.  ;-)

Reply:
If we were to include an example such as https://log.example.com/ct/v2/submit-entry, then what are the chances that some implementers will lazily assume that every log uses "/ct/v2/submit-entry" rather than make the effort of obtaining the actual URL from the log's parameters?

Reply:
I think omitting examples is actually preferable in this case.